integration/feature/cosmo/resources/build.mill

```diff
@@ -0,0 +1,12 @@
+package build
+
+import mill._, javalib._, scalalib._
```

Comment:
Let's put an alias `javalib.cosmo` so people don't need to import it from `scalalib._` in Java projects